Queueomatic/Client/Components/Login/LoginComponent.razor

```diff
@@ -0,0 +1,46 @@
+﻿@using Blazored.SessionStorage
+@inject HttpClient _httpClient;
+@inject NavigationManager _navigationManager;
```

Comment:
These fields are not private. Change their naming convention

Reply:
What naming convention are you suggesting? 

I think this is more readable. These work as fields and it's easier to read and understand.

Reply:
They are properties, and should be treated as such. The are usually capitalized on their first letter.